<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneFolderDialogBase</class>
 <widget class="QDialog" name="CloneFolderDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapHub: Clone Folder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>Clone a folder from MapHub to your local machine.
This will download all maps in the folder and create a QGIS project.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_path">
     <item>
      <widget class="QLabel" name="label_path">
       <property name="text">
        <string>Destination Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_path">
       <property name="placeholderText">
        <string>Select destination directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_browse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_crs">
     <item>
      <widget class="QLabel" name="label_crs">
       <property name="text">
        <string>CRS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsProjectionSelectionWidget" name="projectionSelector"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>CloneFolderDialogBase</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>CloneFolderDialogBase</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>